src/components/Dialog/FooterWithDoubleConfirmation.stories.tsx

```diff
@@ -50,21 +50,22 @@ storiesOf('Components/Dialog/FooterWithDoubleConfirmation', module)
       onCancel={action('onCancel')}
     />
   ))
-  .add('No cancelMessage', () => (
-    <FooterWithDoubleConfirmation
-      mode="normal"
-      valid={false}
-      confirmMessage="Confirm"
-      yesMessage="Yes"
-      noMessage="No"
-      message="Message"
-      onSelectYes={action('onSelectYes')}
-      onSelectNo={action('onSelectNo')}
-      onConfirm={action('onConfirm')}
-      onCancel={action('onCancel')}
-      cancelMessage={undefined}
-    />
-  ))
+  // FIXME: Add `lokiSkip` for this flaky story, as it contains an animation
```

Comment:
@moshensky 
This story is flaky on CI (run fine otherwise), I suppose because of the CSS transition.
I have tried to skip the story using:

```
.storiesOf().add.skip()
or
.addParameters({ component: FooterWithDoubleConfirmation, loki : { skip: true } })
or
.lokiSkip()
```

but I receive TS errors.

WDYT?